paseos/tests/aero_disturbance_test.py

```diff
@@ -0,0 +1,122 @@
+import sys
+
+sys.path.append("../..")
+
+from paseos.attitude.disturbance_torques_utils import compute_aerodynamic_torque
+
+import trimesh
+import numpy as np
+
+
+#  create mesh
+vertices = [
+    [-0.5, -0.5, -0.5],
+    [-0.5, -0.5, 0.5],
+    [-0.5, 0.5, -0.5],
+    [-0.5, 0.5, 0.5],
+    [0.5, -0.5, -0.5],
+    [0.5, -0.5, 0.5],
+    [0.5, 0.5, -0.5],
+    [0.5, 0.5, 0.5],
+]
+faces = [
+    [0, 1, 3],
+    [0, 3, 2],
+    [0, 2, 6],
+    [0, 6, 4],
+    [1, 5, 3],
+    [3, 5, 7],
+    [2, 3, 7],
+    [2, 7, 6],
+    [4, 6, 7],
+    [4, 7, 5],
+    [0, 4, 1],
+    [1, 4, 5],
+]
+mesh = trimesh.Trimesh(vertices, faces)
+spacecraft_temperature_K = 300
+
+
+def test_1():
+    """First test uses existing spacecraft data to check that with the same initial conditions the torque exerted on the
+    satellite is comparable to the real GOCE data. Goce has a cross-section of 1.1 m^2 and during it's drag free
+    phase experienced between 1 and 20 mN of drag force. It is reasonable to expect a torque in the same order of
+    magnitude for a satellite of 1x1x1 m^2 at the same altitude as GOCE.
+    """
+    # GOCE orbital altitude and speed
+    r = np.array([6626, 0, 0])  # [km]
+    v = np.array([0, 7756, 0])  # [m/s]
+
+    T = compute_aerodynamic_torque(
+        r, v, mesh, np.array([np.pi / 4, np.pi / 6, 0]), spacecraft_temperature_K
+    )
+    abs_T = np.linalg.norm(T)
+
+    assert 5 * 10**-4 < abs_T < 2 * 10**-2
+
+
+def test_2a():
+    """Second tests aim to model simple cases and verify that the torque exerted matches the expected torque calculated
+    by the function, with particular emphasis on the torque vector direction.
+
+    Base case: RPY frame is aligned with Body Frame. There should be no torque.
+    """
+    # Circular orbit at GOCE altitude
+    r = np.array([6626, 0, 0])  # [km]
+    v = np.array([0, 7756, 0])  # [km]
+
+    T = compute_aerodynamic_torque(r, v, mesh, np.array([0, 0, 0]), spacecraft_temperature_K)
+
+    assert np.linalg.norm(T) < 10 ** (-8)
```

Comment:
this is not a good test because it just tests is the value is small in enough. It should test against an expected value within some error bounds